challenge_config.yaml

```diff
@@ -37,6 +37,21 @@ challenge_phases:
     max_submissions_per_day: 5
     max_submissions_per_month: 50
     max_submissions: 50
+    submission_meta_attributes:
+      - name: TextAttribute
+        description: Sample
+        type: text
+      - name: SingleOptionAttribute
+        description: Sample
+        type: radio
+        options: ["A", "B", "C"]
+      - name: MultipleChoiceAttribute
+        description: "Sample"

```

Comment:
@KhalidRmb Why is this a string and others are not?

Reply:
Removed the quotes. It's still parsed as a string like other fields.